src/components/postDropdown/container/postDropdownContainer.tsx

```diff
@@ -177,11 +182,137 @@ class PostDropdownContainer extends PureComponent {
           },
         });
         break;
+      case 'mute':
+        this._handleMuteUnmuteUser(!isMuted);
+        break;
+      case 'unmute':
+        this._handleMuteUnmuteUser(!isMuted);
+        break;
       default:
         break;
     }
   };
 
+  _handleMuteUnmuteUser = async (isMuteAction) => {
+    const { currentAccount, content } = this.props as any;
```

Comment:
Needs to handle logged out state somewhere..

<img width="219" alt="Screenshot 2022-08-27 at 12 23 53 PM" src="https://user-images.githubusercontent.com/6298342/187019865-689c4f62-0b64-4821-949e-5815d5e7a1f4.png">